packages/gatsby-theme-docs/src/utils/markdown-fragment-to-react.js

```diff
@@ -33,13 +33,24 @@ const MarkdownFragmentImage = () => null;
  * @param {String} markdownString
  * @param {Object} customElements
  */
-const removeFrontmatter = () => (tree) =>
+const removeFrontmatterPlugin = () => (tree) =>
   filter(tree, (node) => node.type !== 'yaml');
-const markdownFragmentToReact = (markdownString, customElements) =>
-  unified()
+const noOpPlugin = () => (ast) => ast;
+const markdownFragmentToReact = (
+  markdownString,
+  customElements,
+  customPlugin
+) => {
+  const safeCustomPlugin =
+    typeof customPlugin === 'function' ? customPlugin : noOpPlugin;
+  // DEBUG, REMOVE BEFORE FINALIZING
+  console.log(markdownString);
+  console.log(safeCustomPlugin);
```

Comment:
`console.logs` still there

Reply:
OMG, did not push the last commits. Thank you!